lib/resize.js

```diff
@@ -429,7 +429,7 @@ function extract (options) {
 }
 
 /**
- * Trim "boring" pixels from all edges that contain values similar to the top-left pixel.
+ * Trim pixels from all edges that contain values similar to the given background colour, which defaults to that of the top-left pixel.
```

Comment:
Thanks for updating the docs, please can you add an `@example` of providing a colour.

Reply:
Will do